http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -333,21 +263,29 @@ class ClusterMonitorSupervisor(
       .flatMap {
         case Right(clusters) =>
           val clustersNotAlreadyBeingMonitored = clusters.filterNot(c => monitoredClusterIds.contains(c.id))
+          clustersNotAlreadyBeingMonitored.toList
+            .filterA { c =>
+              RuntimeConfigQueries
+                .getRuntimeConfig(c.runtimeConfigId)
+                .transaction
+                .map(x => x.cloudService == CloudService.Dataproc) //only monitor dataproc runtimes
```

Comment:
Could this filtering be done in the DB query (`clusterQuery.listMonitoredClusterOnly`)?

Reply:
updated